faqs/gtn/using-answer-key-histories.md

```diff
@@ -1,20 +0,0 @@
----
```

Comment:
Isn't that FAQ still useful to have?

Reply:
Yes - except this one was better (https://training.galaxyproject.org/training-material/faqs/gtn/gtn_example_histories.html) so I set up a redirect

Reply:
Beating software smog! Woo! Unifying two FAQ into 1! Woo!